app/grouping/create/components/InputForm.client.tsx

```diff
@@ -72,70 +33,17 @@ const inputDefaultValues = {
     toMin: '00',
     toAmPm: 'AM',
   },
-  meetingLocation: '경희대학교',
-  // meetingNumber: 0,
+  meetingLocation: '',
+  meetingNumber: 0,
 };
 
-function getDayName(dayIndex: number) {
-  const days = ['SUN', 'MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT'];
-  return days[dayIndex];
-}
-
-function getMonthName(monthIndex: number) {
-  const months = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'];
-  return months[monthIndex];
-}
-
 export default function InputForm() {
   const imgRef = useRef<HTMLInputElement | null>(null);
   const { register, watch, handleSubmit, setValue } = useForm<InputType>({
     defaultValues: inputDefaultValues,
   });
 
-  const { isModalOpen, openModal, closeModal, modalName } = useModal<ModalNameType>();
-
-  const handlePreviousButton = () => {
-    switch (modalName) {
-      case 'meetingNumber':
-        openModal('meetingLocation');
-        break;
-      case 'meetingLocation':
-        openModal('meetingDate');
-        break;
-      default:
-        console.log('No matching modalName found');
-    }
-  };
-
-  const handleNextButton = () => {
-    switch (modalName) {
-      case 'meetingDate':
-        openModal('meetingLocation');
-        break;
-      case 'meetingLocation':
-        openModal('meetingNumber');
-        break;
-      case 'meetingNumber':
-        if (watch('meetingNumber') === undefined) setValue('meetingNumber', 1);
-        closeModal();
-        break;
-      default:
-        closeModal();
-    }
-  };
-
-  function displayDate() {
-    const date = watch('date');
-    const year = date.getFullYear();
-    const month = getMonthName(date.getMonth());
-    const day = date.getDate();
-    const dayName = getDayName(date.getDay());
-    return `${year}. ${month}. ${day} ${dayName} ${watch('time').fromHour}:${
-      watch('time').fromMin
-    } ${watch('time').fromAmPm} ~ ${watch('time').toHour}:${watch('time').toMin} ${
-      watch('time').toAmPm
-    }`;
-  }
+  const { openModal, closeModal, modalName } = useModal<ModalNameType>();
 
   const handleSubmitButton = (data: InputType) => {
```

Comment:
이 Form 컴포넌트에는 하나의 Submit만 존재하기 때문에 `handleSubmit`으로 핸들러 명을 정하는게 좋아보여요!

```suggestion
  const handleSubmit = (data: InputType) => {
```

Reply:
`handleSubmit`은 `react-hook-form`의 `useModal`훅에서 가져온 함수와 이름이 동일해서 그 부분은 안될 것 같아요 !

Reply:
그러면 저번에 컨벤션 맞춘대로 `handle[~~]Submit`로 하는거 어떤가요?

Reply:
괜찮은 거 같아요 !

제출하는 값들의 Type명이 `CreateMeetingType`이므로 통일성 있게 `handleCreateMeetingSubmit`로 수정했습니다.